terms.html

```diff
@@ -0,0 +1,104 @@
+<p>
+This document attempts to communicate the concepts outlined in the
+<a>decentralized identifier</a> space by using specialized terms to discuss specific
+concepts. This terminology is included below and linked to throughout the
+document to aid the reader:
+</p>
+
+<dl class="termlist">
+
+  <dt><dfn data-lt="DID|DIDs">decentralized identifier</dfn> (DID)</dt>
+
+  <dd>
+A globally unique identifier that does
+not require a centralized registration authority because it is
+registered with <a>distributed ledger technology</a> (DLT) or other form of
```

Comment:
Do we want to rephrase that anchoring a DID to a distributed ledger is optional, for example did:key did:nacl did:peer are more of ledger did's?